conf/messages.cy

```diff
@@ -260,6 +260,7 @@ label.help_improve_gov_uk=Helpwch i wella gwasanaethau CThEM
 label.get.involved.in.making.gov.services.better=Gallwch fod yn rhan o wneud gwasanaethau’r Llywodraeth yn well (yn agor ffenestr neu dab newydd)
 label.no_thanks=Dim diolch
 label.url_coronavirus=Coronafeirws (COVID-19): yr hyn y mae angen i chi ei wneud (yn agor ffenestr neu dab newydd)
```

Comment:
Remove this as its overridden below

Reply:
i kept the original so when we do get translation, we only have to replace "guidance and support" bit and dont have to look for welsh for "opens in a new window or tab"